File: Data Structures & Algorithms/concatenation-of-array/submission-3.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        final int n = nums.length;
        final int[] nums2 = new int[n * 2];

        for (int i = 0; i < n * 2; i++) {
            final int index = i % n;
            nums2[i] = nums[index];
        }

        return nums2;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-3.kt
class Solution {
    fun findMaxConsecutiveOnes(nums: IntArray): Int {
        
        var maximumOnes = 0
        var result = 0

        nums.forEach {
            if (it == 1) maximumOnes += 1
            else { 
                maximumOnes = 0
            }
            result = Math.max(result, maximumOnes)
        }

        return result
    }
}
